sites/cheerpx/src/content/docs/11-guides/File-System-support.md

```diff
@@ -32,9 +32,37 @@ const cx = await CheerpX.Linux.create({
 });
 ```
 
-This mounts the specified local directory at `/web` in the CheerpX filesystem.
+This will mount the specified directory at `/web` in the CheerpX filesystem. To enable listing the directory contents, CheerpX looks for an `index.list` file in the directory and any subdirectories. This file should contain a newline-separated list of all files and subdirectories within the directory.
 
-To be able to list the files, CheerpX will look for a file called index.list in the directory and each of its subdirectories. The file should contain a newline separated list of all files and folders contained in the directory.
+### Why use an `index.list` File?
```

Comment:
Please do not capitalize words that are not the first word or aren't proper names. Please apply this correction to all the titles and subtitles of this guide.